src/FinalRound.js

```diff
@@ -1,14 +1,25 @@
 import Round from './Round';
+import domUpdates from './domUpdates';
 
 class FinalRound extends Round {
   constructor(randomCategories, allPlayers, data) {
     super(randomCategories, allPlayers, data);
+    this.finalClue = this.chooseFinalQuestion();
   }
 
   chooseFinalQuestion() {
     return this.categoryClues[0][Math.floor(Math.random() * 4)];
   }
 
+  validateFinalRoundQuestion(player, playerInput) {
+    if (playerInput.toLowerCase() === this.finalClue.answer.toLowerCase()) {
+      player.score += player.finalWager;
+    } else {
+      player.score -= player.finalWager;
+    }
+    domUpdates.updateScore(player);
```

Comment:
Good extraction of code for this line. You could have put this in each if/else condition, but you moved it outside because it happens for both cases, which is great.